SYCL/Basic/accessor/accessor.cpp

```diff
@@ -827,6 +878,73 @@ int main() {
     }
   }
 
+  // Test byte_size(), size(), max_size(), empty()
+  {
+    std::vector<char> vecChar(64, 'a');
+    std::vector<int> vecInt(32, 1);
+    std::vector<float> vecFloat(16, 1.0);
+    std::vector<IdxID3> vecCustom(8, {0, 0, 0});
+    TestAccSizeFuncs(vecChar);
+    TestAccSizeFuncs(vecInt);
+    TestAccSizeFuncs(vecFloat);
+    TestAccSizeFuncs(vecCustom);
+  }
+  // Test swap() on host_accessor
+  {
+    std::vector<int> vec1(8), vec2(16);
+    try {
+      sycl::buffer<int> buf1(vec1.data(), vec1.size());
+      sycl::buffer<int> buf2(vec2.data(), vec2.size());
+      auto acc1 = buf1.get_access<sycl::access::mode::read_write>();
+      auto acc2 = buf2.get_access<sycl::access::mode::read_write>();
+      acc1.swap(acc2);
+      acc1[15] = acc2[7] = 4;
+    } catch (sycl::exception &e) {
+      std::cout << e.what() << std::endl;
+    }
+    assert(vec1[7] == 4 && vec2[15] == 4);
+  }
+  // Test swap() on basic accessor
+  {
+    std::vector<int> vec1(8), vec2(16);
+    try {
+      sycl::buffer<int> buf1(vec1.data(), vec1.size());
+      sycl::buffer<int> buf2(vec2.data(), vec2.size());
+      sycl::queue q;
+      q.submit([&](sycl::handler &cgh) {
+        auto acc1 = buf1.get_access<sycl::access::mode::read_write>(cgh);
+        auto acc2 = buf2.get_access<sycl::access::mode::read_write>(cgh);
+        acc1.swap(acc2);
+        cgh.single_task([=]() { acc1[15] = acc2[7] = 4; });
+      });
+    } catch (sycl::exception &e) {
+      std::cout << e.what() << std::endl;
+    }
```

Comment:
I don't see a need for `try`/`catch`, same below.

Reply:
Hmm, I believe all these blocks are redundant since no one here can throw an exception, [97607ba](https://github.com/intel/llvm-test-suite/pull/1265/commits/97607baf3d2b37a18bbae1633c990df0d33d74d0)